app/models/user.rb

```diff
@@ -1149,4 +1151,18 @@ def set_avatar_changed
     def reset_avatar_changed
       self.avatar_changed = false
     end
+
+    def name_contains_no_html_tags
```

Comment:
To keep the model cleaner, would you be able to move this to `app/modules/user/validations.rb` as well? (no need to change the tests)

Reply:
that should be it, thank you 👍🏼